### ISSUE TITLE
Warnings in `titanic` and `castle` documentation

### ISSUE THREAD

**Issue by malcolmbarrett** [NickCH-K/causaldata#2]
I believe the `titanic` and `castle` datasets are documented incorrectly in the R package. Per R CMD Check:

```
  Data codoc mismatches from documentation object 'castle':
  Variables in data frame 'castle'
    Code: assault blackm_15_24 blackm_25_44 burglary homicide jhcitizen_c
          jhpolice_c l_exp_pubwelfare l_exp_subsidy l_homicide l_income
          l_lagprisoner l_larceny l_motor l_police l_prisoner lag0 lag1
          lag2 lag3 lag4 lag5 larceny lead1 lead2 lead3 lead4 lead5 lead6
          lead7 lead8 lead9 motor murder popwt poverty r20001 r20002
          r20003 r20004 r20011 r20012 r20013 r20014 r20021 r20022 r20023
          r20024 r20031 r20032 r20033 r20034 r20041 r20042 r20043 r20044
          r20051 r20052 r20053 r20054 r20061 r20062 r20063 r20064 r20071
          r20072 r20073 r20074 r20081 r20082 r20083 r20084 r20091 r20092
          r20093 r20094 r20101 r20102 r20103 r20104 robbery robbery_gun_r
          sid trend_1 trend_10 trend_11 trend_12 trend_13 trend_14
          trend_15 trend_16 trend_17 trend_18 trend_19 trend_2 trend_20
          trend_21 trend_22 trend_23 trend_24 trend_25 trend_26 trend_27
          trend_28 trend_29 trend_3 trend_30 trend_31 trend_32 trend_33
          trend_34 trend_35 trend_36 trend_37 trend_38 trend_39 trend_4
          trend_40 trend_41 trend_42 trend_43 trend_44 trend_45 trend_46
          trend_47 trend_48 trend_49 trend_5 trend_50 trend_51 trend_6
          trend_7 trend_8 trend_9 unemployrt whitem_15_24 whitem_25_44
          year
    Docs: assault blackm_15_24 blackm_25_44 burglary homicide jhcitizen_c
          jhpolice_c l_exp_pubwelfare l_exp_subsidy l_homicide l_income
          l_lagprisoner l_larceny l_motor l_police l_prisoner larceny
          lead1-lag5 motor murder popwt poverty r20001-r20104 robbery
          robbery_gun_r sid trend_1-trend_51 unemployrt whitem_15_24
          whitem_25_44 year
  
  Data codoc mismatches from documentation object 'titanic':
  Variables in data frame 'titanic'
    Code: age class sex survived
    Docs: age any distvct got hiv2004 tinc villnum
```

**Comment by NickCH-K** [comment]
Thanks! Shoot, I was hoping it would let the exclusion of the long list of dummies slide. I'll have to do them another way. And it looks like hte wrong data set entirely was saved as titanic

**Comment by NickCH-K** [comment]
Fixed with 5b8f335
